shipyard_blueprints/box/shipyard_box/cli/download.py

```diff
@@ -43,24 +43,12 @@ def get_args():
     return parser.parse_args()
 
 
-def set_environment_variables(args):
```

Comment:
is not necessary. The application already passes in BOX_APPLICATION_CREDENTIALS as args.service account